app/models/registration/Address.scala

```diff
@@ -39,22 +39,22 @@ object Address {
     countryCode = businessAddress.country.code,
   )
 
-  def fromUkAddress(address: UkAddress): Address = Address(
-    addressLine1 = address.line1,
-    addressLine2 = address.line2,
-    addressLine3 = Some(address.town),
-    addressLine4 = address.county,
-    postalCode = Some(address.postCode),
-    countryCode = address.country.code
+  def apply(ukAddress: UkAddress): Address = Address(
+    addressLine1 = ukAddress.line1,
+    addressLine2 = ukAddress.line2,
+    addressLine3 = Some(ukAddress.town),
+    addressLine4 = ukAddress.county,
+    postalCode = Some(ukAddress.postCode),
+    countryCode = ukAddress.country.code
   )
 
-  def fromJerseyGuernseyIoMAddress(address: JerseyGuernseyIoMAddress): Address = Address(
-    addressLine1 = address.line1,
-    addressLine2 = address.line2,
-    addressLine3 = Some(address.town),
-    addressLine4 = address.county,
-    postalCode = Some(address.postCode),
-    countryCode = address.country.code
+  def fromJerseyGuernseyIoMAddress(jerseyGuernseyIoMAddress: JerseyGuernseyIoMAddress): Address = Address(
```

Comment:
Should we also rename fromJerseyGuernseyIoMAddress to apply and fromInternationalAddress to apply to be consistent  ?

Reply:
I thought about it too. I only changed it for one of the cases. I can change this.